templates/default/client.rb.erb

```diff
@@ -8,6 +8,11 @@
 end
 
 <% end -%>
+
```

Comment:
One other thing. This is going to result in extra spaces being added even if the config isn't templated out which causes updates of every chef run when someone bumps their cookbook. People tend to freak out about that so puttings the spacing inside the unless is helpful here.